### ISSUE TITLE
No keyboard accessibility support for "What lessons..." section

### ISSUE THREAD

**Issue by logan-reynolds** [deraadt/moodle-block_completion_progress#52]
The "What lessons have I viewed for this course?" section is not keyboard accessible. While mouse users are able to move focus between each individual lesson, keyboard users cannot do this. They will be unable to pull up the additional information that appears when users hover over a specific lesson. This will also mean that these users cannot use this section to quickly navigate to a specific lesson.

Each of these elements could be given a tabindex="0" attribute so that they will receive focus. Any device specific event handlers should be replaced to allow keyboard commands to accomplish the same task. For example, since onmouseover is used, the completion information will only update when a mouse is used, regardless of keyboard focus.
